bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "slashroots-foundation",
+  "version": "0.5.0",
+  "homepage": "https://github.com/slashroots/slashroots-website-v2",
+  "authors": [
+    "tremainebuchanan <tremainekbuchanan@gmail.com>"
+  ],
+  "description": "SlashRoots Foundation dynamic website",
+  "main": "keystone.js",
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "angular": "~1.4.8",
```

Comment:
Just something to point out... we should be building against a fixed version of angular and other dependencies.  `~` is an instruction to pull a version reasonably close to the version specified.